### PR TITLE
Add format-js to scripts README for Prettier support

### PR DESCRIPTION
**Pull request by mkaz**
## Description

Updates the scripts README with `format-js` instructions for Prettier support.
This should either be cherry picked into #18048 or merged after.

## Types of changes

Documentation, confirm it works as expected.

### CODE REVIEW

**Review by gziolo — APPROVED** [review]
Awesome work @mkaz. I’d appreciate sanity check from @jsnajdr.

File: packages/scripts/README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ _Example:_
 		"build": "wp-scripts build",
 		"check-engines": "wp-scripts check-engines",
 		"check-licenses": "wp-scripts check-licenses",
+		"format-js": "wp-scripts format-js",
```
Comment (ntwb) [inline packages/scripts/README.md]:
```suggestion
		"format:js": "wp-scripts format-js",
```

Should we keep the naming consistent with the `lint:xyz` nomenclature?

File: packages/scripts/README.md [hunk 2]
```diff
@@ -168,6 +169,31 @@ In the `wp-env` config block, each entry can be configured like so:
 - `test-php`: Runs your plugin's PHPUnit tests. You will need to have an appropriately configured `phpunit.xml.dist` file.
 - `docker-run`: For more advanced debugging, contributors may sometimes need to run commands in the Docker containers. This is the equivalent of running `docker-compose run` within the WordPress directory.
 
+### `format-js`
```
Comment (ntwb) [inline packages/scripts/README.md]:
```suggestion
### `format:js`
```

Should we keep the naming consistent with the lint:xyz nomenclature?

File: packages/scripts/README.md [hunk 3]
```diff
@@ -168,6 +169,31 @@ In the `wp-env` config block, each entry can be configured like so:
...
+
+Formats JavaScript files to the expected coding guidelines.
+
+_Example:_
+
+```json
+{
+	"scripts": {
+		"format:js": "wp-scripts format-js"
```
Comment (ntwb) [inline packages/scripts/README.md]: Hehehe, it already is `format:js` here 😏

**Review by ntwb — CHANGES_REQUESTED** [review]
Looks good, just need to update the instances of `format-js` to `format:js` in line with the `lint:*` nomenclature.

There are already a couple of instances where `format:js` was used to

File: packages/scripts/README.md [hunk 4]
```diff
@@ -169,7 +169,7 @@ In the `wp-env` config block, each entry can be configured like so:
 - `test-php`: Runs your plugin's PHPUnit tests. You will need to have an appropriately configured `phpunit.xml.dist` file.
 - `docker-run`: For more advanced debugging, contributors may sometimes need to run commands in the Docker containers. This is the equivalent of running `docker-compose run` within the WordPress directory.
 
-### `format-js`
+### `format:js`
```
Comment (ntwb) [inline packages/scripts/README.md]: I think this should have been left as is, similar to the other docs sections e.g `### `lint-js``

File: packages/scripts/README.md (hunk 1 above)
Comment (ntwb) [reply packages/scripts/README.md]: This change should be made IMHO @mkaz

File: packages/scripts/README.md (hunk 4 above)
Comment (mkaz) [reply packages/scripts/README.md]: :-)

**Review by ntwb — APPROVED** [review]
👍🏼

**Comment by mkaz** [comment]
Closing in favor of #19946

**Comment by gziolo** [comment]
I see I reviewed it and missed when working on my patch 😞 

The good part is that the version I pushed looks similar :)

**Comment by mkaz** [comment]
I've done similar, so no worries. There's so much happening hard to keep track of every PR. :-)

**Pull request closed without merging**